external/system-test/playlist.xml

```diff
@@ -13,33 +13,30 @@
 # limitations under the License.
 -->
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TKG/playlist.xsd">
-	 <test>
-                <testCaseName>sanity_system</testCaseName>
-                <command>docker run --name sanity_system adoptopenjdk-system-test:latest ; \
-                	$(TEST_STATUS); \
-                	docker rm -f sanity_system; \
-                 	docker rmi -f adoptopenjdk-system-test
-                </command>
-                <levels>
-                	<level>sanity</level>
-                </levels>
-                <groups>
-                 	<group>system</group>
-                </groups>
-        </test>
-
-	 <test>
-                <testCaseName>extended_system</testCaseName>
-                <command>docker run --name extended_system adoptopenjdk-system-test:latest _extended.system; \
-					$(TEST_STATUS); \
-                	docker rm -f extended_system; \
-                 	docker rmi -f adoptopenjdk-system-test
-                </command>
-                <levels>
-                        <level>extended</level>
-                </levels>
-                <groups>
-                        <group>system</group>
-                </groups>
-        </test>
+	<test>
+		<testCaseName>sanity_system</testCaseName>
+		<command> $(TEST_ROOT)$(D)external$(D)external.sh --run --tag ${DOCKERIMAGE_TAG} --version ${JDK_VERSION} --impl ${JDK_IMPL} --dir system-test --reportdst $(REPORTDIR) --docker_args $(EXTRA_DOCKER_ARGS) ; \
```

Comment:
Need to add --testtarget _sanity.system.regular

Reply:
I am not seeing a test target being passed into the old `sanity_system` test. Does it need the `_sanity.system.regular`? Is this something new?